### DISCUSSION TITLE
Why is my Better xCloud menu like this?

### DISCUSSION THREAD

**Discussion by SirERYS** [redphx/better-xcloud#561] [General]
![Screenshot_74](https://github.com/user-attachments/assets/e8f56b71-a9b2-4c35-a757-2c58c964c2be)

Why do the options need to be accessed like this?
This wasn't even bothering me, but when I updated Better xCloud today, I was redirected to the Changelogs page (as usual), and there were screenshots showing the options in the other layout (the style where all the options are displayed on the screen). I prefer that one.
In my Better xCloud, the options are switched by selecting left or right, and that way, I can't see all the options at once.
I hope you understand.

![Screenshot_75](https://github.com/user-attachments/assets/1550dd66-0589-4c6a-b806-7d9f0841c380)

**Comment by redphx** [comment]
Turn off `Controller-friendly UI`

**Reply by SirERYS** [reply]
Okay, I'm almost apologizing for this... how did I not see that option? Actually, I must have thought it was something else, I don't know, but thanks lol. 

I think you're the creator of Better Xcloud. Just know that your plugin is amazing, it enhances Xcloud 100%. There are features I simply can't go without anymore
